src/functions.js

```diff
@@ -3,31 +3,42 @@ function removeMiddle( words ){
   // return a new array containing only the middle word
   // the words array should no longer contain the middle word
   // hint: splice
+  return words.splice(Math.floor(words.length/2), 1);
 }
 
 function get2ndAnd3rd( myArray ){
   // myArray is an array of numbers
   // return an array containing the 2nd and 3rd items from myArray
   // myArray should remain unchanged
   // hint: slice
+  return myArray.slice(1, 3);
 }
 
 function mapper( myArray ){
   // myArray is an array of numbers
   // return a new array which has all items in myArray incremented by one
   // myArray should remain unchanged
   // hint: map
+  var incremented = myArray.map(function(a){
+    return a += 1;
+  });
+  return incremented;
 }
 
 function wordLengths( words ){
   // words is an array of strings
   // return a new array that contains the number of letters in each word
   // hint: strings have a 'length' property
+  var num = [];
+  words.forEach(function(arr){
```

Comment:
Is there a way you could make this more concise? Note that you're taking an existing array and using it to create a new array. This pattern is so common that it's been added as a native array function.

Reply:
yes, done with map method!